modules/apps/asset/asset-list-web/src/main/resources/META-INF/resources/view_asset_list_entry_usages.jsp

```diff
@@ -33,7 +33,9 @@ portletDisplay.setURLBack(redirect);
 renderResponse.setTitle(assetListDisplayContext.getAssetListEntryTitle());
 %>
 
-<div class="container-fluid container-fluid-max-xl container-form-lg">
+<clay:container
+	className="c-pt-mb-5"
```

Comment:
Yeah, this is the one I was referring previously: https://github.com/eduardoallegrini/liferay-portal/pull/70/files#r420811381